keras_gpt_2/model.py

```diff
@@ -77,6 +77,7 @@ def get_model(n_vocab,
               n_embd=768,
               n_head=12,
               n_layer=12,
+              batch=None,
```

Comment:
BTW, I think the argument could be renamed to `batch_size` because Keras uses the name `batch_size` in `fit`.